TM1py/Services/RestService.py

```diff
@@ -654,14 +678,17 @@ def DELETE(
     def logout(self, timeout: float = None, **kwargs):
         """ End TM1 Session and HTTP session
         """
-        # Easier to ask for forgiveness than permission
+
         try:
             # ProductVersion >= TM1 10.2.2 FP 6
             self.POST('/ActiveSession/tm1.Close', '', headers={"Connection": "close"}, timeout=timeout,
                       async_requests_mode=False, **kwargs)
         except TM1pyRestException:
             # ProductVersion < TM1 10.2.2 FP 6
             self.POST('/api/logout', '', headers={"Connection": "close"}, timeout=timeout, **kwargs)
+
+        except requests.exceptions.ConnectionError as ce:
+            print(f"Logout Failed {ce}")
```

Comment:
I think we can change to `warning` here

Reply:
We could use broad except here to cater to all situations where the normal logout fails.

Also, I think we can drop the edge case above for < 10.2.2 FP6